scripts/install-protoc

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
```

Comment:
This is the installer.  There is no way of installing a specific version with apt, so this gets the release direct from google.